query-parser/src/parse.rs

```diff
@@ -163,6 +164,26 @@ def_parser!(Where, or_join, edn::ValueAndSpan, {
     })
 });
 
+def_parser!(Where, not, edn::ValueAndSpan, {
+    satisfy(|v: edn::ValueAndSpan| {
+        if let edn::SpannedValue::PlainSymbol(ref s) = v.inner {
+            s.0.as_str() == "not"
```

Comment:
@ncalexan was going to fix all of these to use `def_matches_plain_symbol`. Please use that for the ones you're adding (see line 359 for an example).